try:
    a = 1 / 0
    print("End")
except Exception as e:
    print("Error!!!: ", e)